packages/instant-apps-components/src/components/instant-apps-app-guide/instant-apps-app-guide.scss

```diff
@@ -1,3 +1,49 @@
 :host {
```

Comment:
Just curious, was there a reason to not use ids/class names for css selectors?

Reply:
Yep, the couple of factors I considered were that web component styles won't get polluted by outside stylesheets, and the simplicity of the component markup made element selectors seem reasonable. 

Per our conversation, as the component evolves and new features are added I will definitely re-evaluate whether this approach needs to change in favor of more specific selectors.  

Also, depending on the particulars of the testing framework, I may need to add classes or IDs to support automated testing (more on that in a future PR).